backend/auth/auth-api/k8/overlays/minikube/secrets.yaml

```diff
@@ -8,8 +8,8 @@ metadata:
     team: auth
 type: Opaque
 stringData:
-  client_id: 237859759623-rfpiq8eo37afp0qc294ioqrjtq17q25h.apps.googleusercontent.com
-  client_secret: 2e4u1VcV24Hi_UsHA0kIJIzr
+  client_id: 237859759623-3skn6doo7bcm1gg6356uf2cq1djrm924.apps.googleusercontent.com
+  client_secret: 06nA-yBVowqETJh5mRRwCCB8
```

Comment:
We want the previous environment to be the default (with redirect to localhost). Those updated one should be in a separate configuration.